webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue

```diff
@@ -62,7 +61,10 @@
                     </div>
                   </td>
                   <td class="center actionContainer">
-                    <i class="uiIconSetting uiIconLightGray"></i>
+                    <i
+                      v-if="!item.isSettingAbsent"
+                      class="uiIconSetting uiIconLightGray"
+                      @click="showSettings(item)"></i>
```

Comment:
Try invoke `provider.showSettings()` directly.

Reply:
It works directly via providerConfig.provider.showSettings()